remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingClient.java

```diff
@@ -321,6 +321,7 @@ public void updateNameServerAddressList(List<String> addrs) {
 
             if (update) {
                 Collections.shuffle(addrs);
+                log.info("name server address updated. NEW : {} , OLD: {}",addrs,old);
```

Comment:
Perhaps we can post another PR for some logging format, currently I would just retain the current way